nearcore/src/config.rs

```diff
@@ -1116,40 +1116,35 @@ pub enum FileDownloadError {
 }
 
 pub fn download_file(url: &str, path: &Path) -> Result<(), FileDownloadError> {
-    return actix::System::new().block_on(async move {
+    let uri = url.parse()?;
+    let (tmp_file, tmp_path) = {
+        let tmp_dir = path.parent().unwrap_or(&Path::new("."));
+        tempfile::NamedTempFile::new_in(tmp_dir).map_err(FileDownloadError::OpenError)?.into_parts()
+    };
+
+    let result = actix::System::new().block_on(async move {
```

Comment:
Unrelated to the PR, but I think it doesn't make sense that we use `actix` here -- it's purely tokio stuff, it doesn't involve neither actix actors, nor actix-web http server. 

I *think* a more frugal way to do this is 

```rust
tokio::runtime::Builder::new_current_thread()
        .enable_all()
        .build()
        .unwrap()
        .block_on(async ...)
```

Reply:
docs: https://tokio.rs/tokio/topics/bridging

Reply:
Hmm, let’s do that in another PR so we have one change per commit.